### ISSUE TITLE
Attempting to run Pokémon Bank results in an error.

### ISSUE THREAD

**Issue by cainan-c** [PabloMK7/citra#47]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

8433057

### Description of Issue

Running any version of Pokémon Bank results in a fatal error, the game is unable to boot. 
Tested on two devices, running two different operating systems.

Was attempting to run it, to see if it'd function with the changes in this fork.

### Expected Behavior

The application should, at least, boot to the menu.

### Reproduction Steps

Run Pokémon Bank

### Log File

[citra_log.txt](https://github.com/PabloMK7/citra/files/14797221/citra_log.txt)

### System Configuration

CPU: Ryzen 7 5800X3D
GPU/Driver: NVIDIA RTX 2060
RAM: 32GB / 3600MHz
OS: Windows 11

CPU: Apple M2 Pro
GPU/Driver: Apple GPU (MoltenVK)
RAM: 16GB
OS: MacOS 14.0 Ventura

**Comment by Defaalt** [comment]
It boots for me. I get an error because I don't have a NNID linked. And I can't figure out how i can import my NNID save. But it boots. Did you decrypt the Cia yourself from your console ?

**Comment by cainan-c** [comment]
Yeah, it's fully decrypted. Re-dumped the titles off of my 3DS to verify that too. Regardless the title fails to boot.
Weirdly Poké Transporter boots fine
